.env

```diff
@@ -0,0 +1 @@
+export SOUNDCLOUD_CLIENT_ID=69e93cf2209402f6f3137a6452cf498f
```

Comment:
... or maybe the acceptance tests would work with any client id.

Also realized that this is the `client_id` and not the `client_secret`.